specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2023-03-01/costmanagement.benefits.json

```diff
@@ -845,35 +1511,148 @@
           "type": "number",
           "format": "decimal",
           "readOnly": true
         },
         "maxUtilizationPercentage": {
-          "description": "This is the maximum hourly utilization for each date range that corresponds to given grain (Daily, Monthly). Suppose the API call is for usageDate > 2022-10-01 and usageDate < 2022-10-31 at a daily granularity. There will be one record per benefit id for each day. For a single day, the maxUtilizationPercentage value will be equal to the largest in the set of values where the set contains 24 utilization percentage entries one for each hour in a specific day. If on the day 2022-10-18, the largest utilization percentage was 90% at hour 5, then the value for the maxUtilizationPercentage in the response will be 90%.",
+          "description": "This is the maximum hourly utilization for each date range that corresponds to given grain (Daily, Monthly). Suppose the API call is for usageDate > 2023-03-01 and usageDate < 2022-10-31 at a daily granularity. There will be one record per benefit id for each day. For a single day, the maxUtilizationPercentage value will be equal to the largest in the set of values where the set contains 24 utilization percentage entries one for each hour in a specific day. If on the day 2022-10-18, the largest utilization percentage was 90% at hour 5, then the value for the maxUtilizationPercentage in the response will be 90%.",
           "type": "number",
           "format": "decimal",
           "readOnly": true
         }
       }
+    },
+    "BenefitUtilizationSummariesOperationStatus": {
+      "type": "object",
+      "description": "Status of a benefit utilization summaries report. Provides Async Benefit Utilization Summaries Request input, status, and report sas url.",
+      "properties": {
+        "input": {
+          "$ref": "#/definitions/BenefitUtilizationSummariesRequest",
+          "description": "Input given to create the benefit utilization summaries report."
+        },
+        "status": {
+          "description": "The status of the creation of the benefit utilization summaries report.",
+          "$ref": "#/definitions/AsyncOperationStatusEnum"
+        },
+        "properties": {
+          "description": "Contains sas url to the async benefit utilization summaries report and a date that the url is valid until. These values will be empty if the report is in a Running or Failed state",
+          "$ref": "#/definitions/AsyncOperationStatusProperties"
+        }
+      }
+    },
+    "AsyncOperationStatusEnum": {
+      "description": "Enum representing the status of an async operation.",
+      "type": "string",
+      "enum": [
+        "Running",
+        "Complete",
+        "Failed"
+      ],
+      "x-ms-enum": {
+        "name": "Status",
```

Comment:
Could you rename this enum as the name is duplicate with a previous model: https://github.com/Azure/azure-rest-api-specs/blob/82c89f9277f470bb26ffe0394628405f122cbf33/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2023-03-01/costmanagement.generatedetailedcostreport.json#L334, also it seems the enum is the same with this one: https://github.com/Azure/azure-rest-api-specs/blob/82c89f9277f470bb26ffe0394628405f122cbf33/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2023-03-01/costmanagement.json#L2637, is it possible to use the same name: OperationStatusType?